node-data/src/message.rs

```diff
@@ -275,34 +226,43 @@ impl Message {
     /// Creates topics.GetData  message
     pub fn new_get_data(p: payload::Inv) -> Message {
         Self {
-            header: Header::new(Topics::GetData),
+            topic: Topics::GetData,
             payload: Payload::GetInv(p),
```

Comment:
I know it's not part of this PR, but it seems that's the wrong payload to set (it's not just about naming). So I pointed it out. I guess it can be addressed in a different PR.

Reply:
@goshawk-3 what do you think about it?

Reply:
> shouldn't this be `Payload::GetData` ?

This replicates bitcoin messages. 
"The format and maximum size limitations of the [“getdata” message](https://developer.bitcoin.org/reference/p2p_networking.html#getdata) are identical to the [“inv” message](https://developer.bitcoin.org/reference/p2p_networking.html#inv); only the message header differs."

src: https://developer.bitcoin.org/reference/p2p_networking.html#getdata

Reply:
Thanks, now it's clear to me. I removed those changes